### ISSUE TITLE
starargs from lambda result in no-value-for-parameter

### ISSUE THREAD

**Issue by aure-olli** [PyCQA/pylint#2918]
### Steps to reproduce
```python
def foo(one, two, three):
    pass

bar = lambda *args: foo(*args)
```

### Current behavior
```
test.py:4:20: E1120: No value for argument 'one' in function call (no-value-for-parameter)
test.py:4:20: E1120: No value for argument 'two' in function call (no-value-for-parameter)
test.py:4:20: E1120: No value for argument 'three' in function call (no-value-for-parameter)
```

### Expected behavior
No warning

### pylint --version output
This is exactly https://github.com/PyCQA/pylint/issues/722, except I'm using a lambda function.

I hope it can be similarly fixed

**Comment by PCManticore** [comment]
Hi, thank you for the report! I can confirm the bug.
